openapi/paths/aml-checks.yaml

```
@@ -0,0 +1,37 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve customer AML checks
```

suggestion: would it make sense to remove `customer` here also?
```suggestion
  summary: Retrieve AML checks
```